crates/ruff_linter/src/settings/mod.rs

```diff
@@ -43,8 +45,7 @@ pub struct LinterSettings {
 
     pub rules: RuleTable,
     pub per_file_ignores: Vec<(GlobMatcher, GlobMatcher, RuleSet)>,
-    pub extend_unsafe_fixes: RuleSet,
-    pub extend_safe_fixes: RuleSet,
+    pub fix_safety_table: FixSafetyTable,
```

Comment:
I think I have a preference for something like `fix_safety` i.e. we don't include `table` for the `RuleTable` field